external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -71,6 +71,14 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
 
   def testCreateTableWithProperty(tbl: String): Unit = {}
 
+  def indexOptions(catalogName: String): String = {
+    catalogName match {
+      case "mysql" => "KEY_BLOCK_SIZE=10"
+      case "postgresql" => "FILLFACTOR=70"
+      case _ => ""
+    }
+  }
```

Comment:
I think what @cloud-fan meant is to add a method such as 
```
def indexOptions: String = ""
```
then in each of the concrete test suite, implement this method, e.g. in `MySQLIntegrationSuite`,  implement it as 
```
override def indexOptions: String = "KEY_BLOCK_SIZE=10"
```

Reply:
Thank you for clarifying.